Makefile

```diff
@@ -1,86 +1,24 @@
 SHELL = /bin/bash
 GO ?= go
 CC ?= gcc
-COVERAGE_PATH ?= $(shell pwd)/.coverage
-CRIU_FEATURE_MEM_TRACK = $(shell if criu check --feature mem_dirty_track > /dev/null; then echo 1; else echo 0; fi)
-CRIU_FEATURE_LAZY_PAGES = $(shell if criu check --feature uffd-noncoop > /dev/null; then echo 1; else echo 0; fi)
-CRIU_FEATURE_PIDFD_STORE = $(shell if criu check --feature pidfd_store > /dev/null; then echo 1; else echo 0; fi)
 
-export CRIU_FEATURE_MEM_TRACK CRIU_FEATURE_LAZY_PAGES CRIU_FEATURE_PIDFD_STORE
-
-all: build test phaul-test
+all: build
 
 lint:
 	golangci-lint run ./...
 
 build:
+	$(Q) $(MAKE) -C scripts
```

Comment:
Do you define `$(Q)` somewhere? I don't think we have the setup in go-criu to support quiet builds.

Reply:
Thank you for pointing this out. I have dropped `$(Q)` and updated the pull request.